CONTRIBUTING.rst

```diff
@@ -13,7 +13,7 @@ To begin contributing have a look at the open issues labeled with
 difficulty/newcomer - `newcomer issues <https://github.com/coala-analyzer/coala/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer>`__.
 
 Follow the instructions
-`here <http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__
+`here <http://coala.readthedocs.io/en/latest/Developers/Newcomers_Guide.html>`__
```

Comment:
just link to `https://coala.io/newcomers`, I made this short version so we can change the URL in one place in case it changes again.